src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { getArticlesData } from "../../app";
+import  ArticleCard  from "./ArticleCard"
+
+
+function ArticleList() {
+
+    const [articles, setArticlesData] = useState([])
+
+    useEffect(() => {
+    getArticlesData()
+    .then((data) => {
+        setArticlesData(data.articles)
```

Comment:
Add an `isLoading` pattern is followed with visual feedback to inform user that data is being fetched.